src/core/models/log.py

```
@@ -70,33 +66,44 @@ def _project_id(self) -> ULID:
     def _task_status(self) -> TaskStatus:
         return TaskStatus(self.task_status)
 
-    def to_dict(self, visited=None):
+    def to_dict(self, visited: set[int] | None = None) -> dict:
```

Ruff (0.14.14)

Unused method argument: `visited`

(ARG002)